docs/design-language.md

```diff
@@ -238,72 +312,68 @@ Selected and Active states for list elements should showcase similar behavior to
+<div class="typography buttons-box">
+    <h1>h1. Heading (2rem)</h1>
+    <h2>h2. Heading (1.812rem)</h2>
+    <h3>h3. Heading (1.625rem)</h3>
+    <h4>h4. Heading (1.438rem)</h4>
+    <h5>h5. Heading (1.250rem)</h5>
+    <h6>h6. Heading (1rem)</h6>
+    <h6 class="subtitle1">subtitle1 (1rem, increased line spacing). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur</h6>
+    <h6 class="subtitle2">subtitle2 (0.875rem). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur</h6>
+    <p class="body1">body1 (0.875rem). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam.</p>
+    <p class="body2">body2 (0.75rem). Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quos blanditiis tenetur unde suscipit, quam beatae rerum inventore consectetur, neque doloribus, cupiditate numquam dignissimos laborum fugiat deleniti? Eum quasi quidem quibusdam.</p>
+    <p class="caption">Caption text (0.875rem)</p>
+    <p class="overline">Overline text (0.75rem)</p>
+</div><br />
 
-Panels come with an assortment of configurable navigation controls:
+### Font Stack
 
-![Panel maximize icon](/img/panels/panel-maximize.jpg "Maximize Button") Maximize: This button will extend the content of the panel to fill the entire view width.
+Our current font stack is <em>"Segoe UI, Helvetica Neue, Roboto, Helvetica, Arial, sans-serif"</em>. **<a href="https://learn.microsoft.com/en-us/typography/font-list/segoe-ui" target="_blank">Segoe UI</a>** is installed by default on Windows. MacOS/iOS will take advantage of Helvitca Neue and Roboto is Android's default font. **Font sizes rely on the browser default font-size of 16px. This value should not be changed.**
 
-![Panel minimize icon](/img/panels/panel-minimize.jpg "Minimize Button") Minimize: The minimize button will collapse the panel down fully. It is expandable again by click the ‘Restore to Original Size’ button that replaces minimize.
+### Headings
 
-![Panel back icon](/img/panels/panel-back.jpg "Back Button") Back Button: Navigates you to the previous panels content.
+Headings are the largest text on the screen and are used for titles. They help organize content and demonstrate importance using a hierarchy that ranges from 1-6.
+<img src="/img/typography/h1.png" alt="H1 as found in VertiGIS Studio Web" title="H1 as found in VertiGIS Studio Web" class="img-example" />
+<br /><br />
 
-![Panel close icon](/img/panels/panel-close.jpg "Close Button") Close Button: Closes the panel entirely.
+#### Considerations when using Headings
 
-### Examples
+-   Do not alter the weight of headings using CSS or HTML tags.
+-   Avoid skipping heading levels. More info can be found at the **<a href="https://www.w3.org/WAI/tutorials/page-structure/headings/" target="_blank">W3C</a>**
```

Comment:
Maybe mention that a page can have at most one H1 heading. The provided link doesn't really mention that, but it's both good practice and mentioned elsewhere in accessibility guidelines.